src/plugins/data_view_editor/public/components/form_fields/type_field.tsx

```diff
@@ -57,6 +58,15 @@ const rollupSelectItem = (
       <EuiBadge color={euiLightVars.euiColorAccent}>
         <FormattedMessage id="indexPatternEditor.typeSelect.betaLabel" defaultMessage="Beta" />
       </EuiBadge>
+      &nbsp;
+      <RollupDeprecationTooltip>
+        <EuiBadge color="warning">
+          <FormattedMessage
+            id="indexPatternEditor.typeSelect.deprecatedBadge"
+            defaultMessage="Deprecated"
+          />
+        </EuiBadge>
+      </RollupDeprecationTooltip>
```

Comment:
Can we also update i18n `indexPatternEditor.typeSelect.rollup` string on  Line 103 with `Rollup (deprecated)`?

Reply:
Thanks for the review @jughosta! Yes, it makes sense to me to rename the rollup option to `Rollup (deprecated)`. Regarding changing the Beta badge color to gray, @jovana-andjelkovic do you have any opinion on this?

<img width="1164" alt="Screenshot 2024-07-02 at 16 42 02" src="https://github.com/elastic/kibana/assets/59341489/54d0d982-d796-48c4-a7f6-37026940c326">

Reply:
I agree with renaming the rollup option as well 👍 

@ElenaStoeva Given that we're using gray color for Index in this context, I would avoid using the same for Beta. 

I took a look a look at [EUI specs](https://eui.elastic.co/#/display/badge#beta-badge-type) and we don't have guidelines on when which version of the Beta badge is used, and it looks like we're not using the designated beta badge here anyways 🤔 

Overall, I do see the point in making Beta badge stand out less, but given the usage of Rollups, I would say **let's keep the pink color as is**.

cc @alisonelizabeth @formgeist @tylerperk - no action needed, just to be aware of changes.